src/main/java/org/ibp/api/rest/study/StudyInstanceResource.java

```diff
@@ -25,35 +28,35 @@
-		notes = "Create new study instance")
+	@ApiOperation(value = "Create new study instances",
+		notes = "Create new study instances")
 	@PreAuthorize("hasAnyAuthority('ADMIN','BREEDING_ACTIVITIES','MANAGE_STUDIES')")
 	@RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/instances/generation", method = RequestMethod.POST)
 	@ResponseBody
-	public ResponseEntity<StudyInstance> createStudyInstance(final @PathVariable String cropname, @PathVariable final String programUUID,
-		@PathVariable final Integer studyId) {
-		return new ResponseEntity<>(this.studyInstanceService.createStudyInstance(cropname, studyId),
+	public ResponseEntity<List<StudyInstance>> createStudyInstances(final @PathVariable String cropname, @PathVariable final String programUUID,
+		@PathVariable final Integer studyId, @RequestParam final Integer numberOfInstancesToGenerate) {
+		return new ResponseEntity<>(this.studyInstanceService.createStudyInstances(cropname, studyId, numberOfInstancesToGenerate),
 			HttpStatus.OK);
 
 	}
 
-	@ApiOperation(value = "Delete study instance",
-		notes = "Delete study instance")
+	@ApiOperation(value = "Delete study instances",
+		notes = "Delete study instances")
 	@PreAuthorize("hasAnyAuthority('ADMIN','BREEDING_ACTIVITIES','MANAGE_STUDIES')")
-	@RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/instances/{instanceId}", method = RequestMethod.DELETE)
+	@RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/instances", method = RequestMethod.DELETE)
 	@ResponseBody
-	public ResponseEntity<Void> deleteStudyInstance(final @PathVariable String cropname, @PathVariable final String programUUID,
-		@PathVariable final Integer studyId, @PathVariable final Integer instanceId) {
-		this.studyInstanceService.deleteStudyInstance(studyId, instanceId);
+	public ResponseEntity<Void> deleteStudyInstances(final @PathVariable String cropname, @PathVariable final String programUUID,
+		@PathVariable final Integer studyId, @RequestParam final List<Integer> instanceIds) {
+		this.studyInstanceService.deleteStudyInstances(studyId, instanceIds);
 		return new ResponseEntity<>(HttpStatus.OK);
 	}
 
 	@ApiOperation(value = "List all study instances with basic metadata.",
 		notes = "Returns list of all study instances with basic metadata.")
-	@PreAuthorize("hasAnyAuthority('ADMIN','BREEDING_ACTIVITIES','MANAGE_STUDIES', 'INFORMATION_MANAGEMENT', 'BROWSE_STUDIES')")
+	@PreAuthorize("hasAnyAuthority('ADMIN','BREEDING_ACTIVITIES','MANAGE_STUDIES')")
 	@RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/instances", method = RequestMethod.GET)
 	@ResponseBody
-	public ResponseEntity<List<StudyInstance>> listStudyInstances(final @PathVariable String cropname, @PathVariable final String programUUID,
+	public ResponseEntity<List<StudyInstance>> listStudyEnvironments(final @PathVariable String cropname, @PathVariable final String programUUID,
```

Comment:
Use `listStudyInstances` to be consistent?

Reply:
Also why did we remove the following authorities from seeing the instances: 'INFORMATION_MANAGEMENT', 'BROWSE_STUDIES'

Reply:
Shiela commented on the ticket that the API should only be accessible by a user with Manage Studies permission.

Reply:
I think we should revert the old behavior to standardize the behavior of other GET endpoints in other study-related resource classes (eg. DatasetResource)